src/main/java/frc/robot/subsystems/arm/ArmSubsystem.java

```diff
@@ -115,4 +115,12 @@ public Command increaseAngle(double degrees) {
   public Command decreaseAngle(double degrees) {
     return increaseAngle(-degrees);
   }
+
+  public Command engageEmergencyMode() {
+    return Commands.runOnce(() -> armIO.setEmergencyMode(true));
+  }
+
+  public Command exitEmergencyMode() {
+    return Commands.runOnce(() -> armIO.setEmergencyMode(false));
```

Comment:
within `setEmergencyMode` in `ArmIOReal`, let's also go ahead and implement a check to see whether it is true or false. If false, then we just need to set the voltage back to the originally requested voltage